src/components/Input/readme.md

```diff
@@ -17,6 +17,33 @@ const containerStyles = {
 />;
 ```
 
+##### input base with an initial value set
```

Comment:
I think we should start adding a useful and short little description for the new interactive examples